src/experiments/webgpuaudio/main.js

```diff
@@ -33,7 +33,7 @@ const createAudioContext = async () => {
   // Initially suspend audioContext so it can be toggled on and off later.
   audioContext.suspend();
   // Start the oscillator
-  oscillator.start();
+  oscillator.start();  
```

Comment:
It's weird that I see this diff. The upstream `main.js` has this line already. Have you rebased your local repository/branch based on the current upstream?

Reply:
Yes this is weird. I just rebased and fixed all merge conflicts.